common.vars.pkr.hcl

```diff
@@ -1 +1 @@
-postgres-version = "15.1.1.90"
+postgres-version = "15.1.1.91"
```

Comment:
I believe we can stop updating this file now, and will work to remove these old files ASAP